spec/list_of_dishes_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'list_of_dishes'
+
+describe ListOfDishes do
+
+  let(:fish_dish) { double("fake_fish") }
+  let(:toast_dish) { double("fake_toast") }
+
+  before(:each) do
+    @list_of_dishes = ListOfDishes.new
+  end
+
+  it "should be an instance of ListOfDishes" do
+    expect(@list_of_dishes).to be_instance_of(ListOfDishes)
+  end
```

Comment:
I know I"m not the expert on this, but is this an example of testing state? You create a thing and then immediately test that it is an instance of that class. Very clear and descriptive names though!